diff --git a/node_modules/unstorage/drivers/vercel-kv.cjs b/node_modules/unstorage/drivers/vercel-kv.cjs
index dc1e6d7..2646392 100644
--- a/node_modules/unstorage/drivers/vercel-kv.cjs
+++ b/node_modules/unstorage/drivers/vercel-kv.cjs
@@ -34,6 +34,7 @@ module.exports = (0, _utils.defineDriver)(opts => {
     return _client;
   };
   return {
+    name: "vercel-kv",
     hasItem(key) {
       return getClient().exists(r(key)).then(Boolean);
     },
diff --git a/node_modules/unstorage/drivers/vercel-kv.mjs b/node_modules/unstorage/drivers/vercel-kv.mjs
index b31610a..95a8d34 100644
--- a/node_modules/unstorage/drivers/vercel-kv.mjs
+++ b/node_modules/unstorage/drivers/vercel-kv.mjs
@@ -34,6 +34,7 @@ export default defineDriver((opts) => {
     return _client;
   };
   return {
+    name: "vercel-kv",
     hasItem(key) {
       return getClient().exists(r(key)).then(Boolean);
     },
